### ISSUE TITLE
wikquote bugs

### ISSUE THREAD

**Issue by cantsin** [gamelost/bot3server#18]
- don't crash on invalid input
- regex only matches the first word

**Comment by cantsin** [comment]
@timkettering would it be feasible/desirable to implement pastebin (or the equivalent) so that it would work for all modules, regardless of their output?

**Comment by timkettering** [comment]
agreed.  should be a lib that can take input, and return a created pastebin link.
